modules/core/04-channel/keeper/handshake_test.go

```diff
@@ -638,7 +638,6 @@ func (suite *KeeperTestSuite) TestChanCloseInit() {
 
 			// create channel in init
 			path.SetChannelOrdered()
-			err = path.EndpointA.ChanOpenInit()
```

Comment:
Why was this removed?

Reply:
I added it back and added as well a`suite.Require().NoError(err)`